salt/docker_apps/files/pelican_frontend.yaml

```
@@ -1,14 +1,11 @@
 version: "2.2"
 
-x-shared: &shared
-  image: "ghcr.io/open-contracting/pelican-frontend-django:latest"
-  network_mode: "host"
-
-x-daemon: &daemon
-  <<: *shared
-  restart: unless-stopped
-
 services:
   django:
-    <<: *daemon
-    command: "./gunicorn_runner.sh {{ pillar.docker_apps.pelican_frontend.port }}"
+    image: "ghcr.io/open-contracting/pelican-frontend-django:latest"
+    restart: unless-stopped
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

nitpick: In other files, restart is declared after network_mode/extra_hosts.

```suggestion
    extra_hosts:
      - "host.docker.internal:host-gateway"
    restart: unless-stopped
```

---

Reordered